frontend/device/remarkable/powerd.lua

```diff
@@ -0,0 +1,30 @@
+local BasePowerD = require("device/generic/powerd")
+
+-- TODO older firmware doesn't have the -0 on the end of the file path
```

Comment:
Probably won't be able to run the binaries because of an older glibc (I presume) anyway ;).